ral-facilities/operationsgateway
# Patching minimatch to v3.1.2

## Description
Following [a dependabot alert](https://github.com/ral-facilities/operationsgateway/security/dependabot/3) and its subsequent failure to update, I have made this PR to update minimatch v3 to the latest patched legacy version. The earliest patched version is v3.0.5. v3.0.4 was still being used in places so this resolves all uses of minimatch v3 to v3.1.2 (latest legacy version)

## Testing instructions
I ran the command:
```
yarn set resolution minimatch@npm:^3.0.4 3.1.2
```
Check the changes made to yarn.lock. They should ensure that the resolution is correct.


## Agile board tracking
Hotfix
